zerver/management/commands/delete_old_unclaimed_attachments.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from zerver.lib.actions import do_delete_old_unclaimed_attachments
+from zerver.models import Attachment, get_old_unclaimed_attachments
+
+class Command(BaseCommand):
+    help = """Remove unclaimed attachments from storage older than a supplied
+              numerical value indicating the limit of how old the attachment can be.
+              One week is taken as the default value."""
+
+    def add_arguments(self, parser):
+        parser.add_argument('-w', '--weeks',
+                            dest='delta_weeks',
+                            default=1,
+                            help="Limiting value of how old the file can be.")
+
+        parser.add_argument('-f', '--for-real',
+                            dest='for_real',
+                            action='store_true',
+                            default=False,
+                            help="Actually remove the files from the storage.")
+
+    def handle(self, *args, **options):
+
+        delta_weeks = options['delta_weeks']
+        print("Deleting files older than %s" % (delta_weeks,))
```

Comment:
Should be clearer that only some files are being deleted, e.g. "Deleting unclaimed attached files older than %s"